packages/hunspell.rb

```diff
@@ -37,14 +31,18 @@ def self.build
 
   def self.install
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+  end
+
+  def self.postinstall
     system 'git clone -b libreoffice-6-1 --depth 1 git://anongit.freedesktop.org/libreoffice/dictionaries.git'
     system "install -Dm644 dictionaries/en/en_US.aff #{CREW_DEST_DIR}$HOME/Library/Spelling/en_US.aff"
     system "install -Dm644 dictionaries/en/en_US.dic #{CREW_DEST_DIR}$HOME/Library/Spelling/en_US.dic"
     system "install -Dm644 dictionaries/en/en_US.aff $HOME/Library/Spelling/en_US.aff"
     system "install -Dm644 dictionaries/en/en_US.dic $HOME/Library/Spelling/en_US.dic"
-  end
-
-  def self.postinstall
+    system "install -Dm644 dictionaries/fr_FR/fr.aff #{CREW_DEST_DIR}$HOME/Library/Spelling/fr_FR.aff"
+    system "install -Dm644 dictionaries/fr_FR/fr.dic #{CREW_DEST_DIR}$HOME/Library/Spelling/fr_FR.dic"
+    system "install -Dm644 dictionaries/fr_FR/fr.aff $HOME/Library/Spelling/fr_FR.aff"
+    system "install -Dm644 dictionaries/fr_FR/fr.dic $HOME/Library/Spelling/fr_FR.dic"
```

Comment:
How could we allow users to install their own dictionaries ? Is it part of these packages ? This is an easy solution but probably not the cleanest one especially in term of separation of concerns

Reply:
This is actually a great way to separate dictionary updates from the package.  We can run `crew postinstall hunspell` to perform the updates.  We might want to add a postinstall message to explain:
```
puts "To update the dictionaries periodically, execute 'crew postinstall hunspell'.".lightblue
```

Reply:
We can just update this package when someone wants to add another language.

Reply:
The side effect is that one could have a large amount of installed files he doesn't need : for instance here, if one doesn't need french dictionaries, he will be forced to install them

Reply:
We could add languages in separate packages like we do with aspell.

Reply:
Why not ! No install step, just a postinstall one to allow manual periodically updates ? Is there a stub for `version`, `source_url` and `source_sha256` ?

Reply:
We could keep en_US by default in hunspell

Reply:
Example given in 58ec722

Reply:
There is `template.rb.template` but it might be easier just to copy hunspell.rb and modify.

Reply:
Isn't there a way to download absolutely nothing ?

Reply:
There is this but I don't like it since it doesn't uniquely identify the package and can cause a conflict with other packages that are being installed at the same time.
```
  source_url 'file:///dev/null'
  source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
```

Reply:
OK. Let's keep the source_url to hunspell.tar.gz so.